router/core/context.go

```diff
@@ -266,6 +277,15 @@ func (c *requestContext) GetStringMapStringSlice(key string) (smss map[string][]
 	return
 }
 
+func (c *requestContext) ActiveSubgraph(subgraphRequest *http.Request) *Subgraph {
+	for _, sg := range c.subgraphs {
+		if sg.Url == subgraphRequest.URL.String() {
```

Comment:
Should we normalize the request URL before comparison? Host:port match should be enough.

Reply:
We do it on the control plane already. But updated comparison here anyways